index.html

```diff
@@ -265,5 +265,43 @@ <h3>Skills</h3>
         </div>
       </section>
     </section>
+
+    <!-- Contact section -->
+    <footer class="contact">
+      <h1>Contact Me</h1>
+      <p>
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/mvolzpoz" method="post" id="my-form">
+        <input
+          type="text"
+          placeholder="Full name"
+          maxlength="30"
+          name="full-name"
+          required
+        />
+        <input type="email" placeholder="Email address" name="email" required />
```

Comment:
- [ ] Let's make sure we follow the Figma design, looks like here we have an undesired border around our inputs/buttons, can we fix that, please? 

Your Implementation | Figma Design
:-----:|:------:
![image](https://user-images.githubusercontent.com/87689547/148969662-3f364516-0c08-47e3-85e1-dd83a7e61304.png) | ![image](https://user-images.githubusercontent.com/87689547/148970137-b90de1ce-3d7b-413d-b289-6e2139126d0c.png)